### PR TITLE
Feature/issue 4 implement content negotiations

### PR DESCRIPTION
**Pull request by bsse1009**
### Feature: Content Negotiation

**Content-type:**

- XML
- TEXT
- HTML
- JSON

### CODE REVIEW

File: server/utils/contentNegotiation.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use strict'
+const js2xmlparser = require("js2xmlparser");
+const json2html = require('json-to-html');
+const {toPlainText} = require('json-to-plain-text');
+
+class ContentNegotiation{
+    constructor(res, statuscode, data) {
```
Comment (rafaelragib) [inline server/utils/contentNegotiation.js:7]:
```suggestion
    constructor(res, statusCode, data) {
```
status code is two words. use camelcase for multiple words

File: server/controllers/stories.controller.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
 "use strict";
 const {StoriesService} = require('../services');
+const {ConstentNegotiation} = require('../utils');
```
Comment (rafaelragib) [inline server/controllers/stories.controller.js:3]: fix name

**Review by rafaelragib — APPROVED** [review]
Fix the name and merge the PR. I am approving it.

File: server/controllers/stories.controller.js (hunk 2 above)
Comment (bsse1009) [reply server/controllers/stories.controller.js:3]: was fixed in latest commit.

File: server/utils/contentNegotiation.js (hunk 1 above)
Comment (bsse1009) [reply server/utils/contentNegotiation.js:7]: ok

**Pull request merged**